### ISSUE TITLE
Judge/CA Dashboard: Pending Motions

### ISSUE THREAD

**Issue by cholly75** [ustaxcourt/ef-cms#8528]
As a judge or CA, so that I can have a more complete picture of my overall activity and outstanding items to be addressed, I need a Pending Motions interface added to my dashboard.

With the success of the Activity report in providing a view of judge/chambers workflow, the Court has requested additional data to track in the same vein. The current request is to provide a subset of the information currently seen on the Pending Report in a similar UI to the Submitted/CAV cases portion of the judge/chambers dashboard. Instead of capturing all pending items as per the Pending Report, this subset will focus specifically on Motions that have been Pending for 6 months or more.

Information from the Pending Motions table will be added as a companion segment on the Activity report in a future story.

## Acceptance Criteria
* On the Judge/Chambers dashboard, a new section underneath the Submitted/CAV case table labeled “Pending Motions”
  - ✔️ This table has the following properties:
    - ✔️ Reflects all pending motions on cases assigned to a judge where the motion has been a Pending Item on the case for 180 days or greater
    - ✔️ Each row on the table is an individual motion
    - ✔️ In dockets where multiple motions meet the criteria for inclusion, each motion is shown on its own separate row
    - ✔️ Motions multidocketed across cases in a consolidated group are represented singly on the lead case
    - ✔️ Motions filed singly in member cases are represented singly on the member case
    - ✔️ Motions filed singly in a Lead case are represented singly on the Lead case
* ✔️ Each row on the table contains the following information:
  - ✔️ Docket # (with suffix and appropriate consolidated case icons), No. of Cases, Petitioners, "generic" title of motion type (as opposed to customized description seen on the docket record), # of days pending
* ✔️ Each row on the table provides a method for the user to enter contextual information in the same manner as the rows in the Submitted/CAV Cases table
  - ✔️ Primary Issue, Final Brief Due Date, Status of Matter
* ✔️ Rows on the table are sorted by # of days pending, most to least
* Removal of a motion from the Pending Items tab on the case removes it from the Pending Motions table

## Notes
* Desire is for any given document ID to show only once on the report
* 40-50 Pending Motions > 180 days in Pending is expected to be the high end in production
* Motions removed from the Pending Report for a case can be added back if the motion docket entry is edited; ~~in these cases we would re-start the "pending" clock~~. We do not need to retain previously entered contextual information for motions when this occurs.

## Story Definition of Ready (updated on 12/23/22)
The following criteria must be met in order for the user story to be picked up by the Flexion development team.
The user story must: 
- [x] Is framed in business/user need, the value has been addressed.
- [ ] Includes acceptance criteria
- [ ] Has been refined
- [ ] Pre conditions have been satisfied. 

**Process:** 
Flexion developers and designers will test if the story meets acceptance criteria and test cases in Flexion dev and staging environments (“standard testing”). If additional acceptance criteria or testing scenarios are discovered while the story is in progress, a new story should be created, added to the backlog and prioritized by the product owner. 

## Definition of Done (Updated 5-19-22)
**Product Owner**
 - [ ]  Acceptance criteria have been met and validated on the Court's migration environment
 - [ ] Add scenario to testing document, if applicable (https://docs.google.com/spreadsheets/d/1FUHKC_YrT-PosaWD5gRVmsDzI1HS_U-8CyMIb-qX9EA/edit?usp=sharing)

**UX**
 - [ ] Business test scenarios have been refined to meet all acceptance criteria 
 - [ ] Usability has been validated
 - [ ] Wiki has been updated (if applicable) 
 - [ ] Story has been tested on a mobile device (for external users only)

**Engineering**
 - [x] Automated test scripts have been written, including visual tests for newly added PDFs.
 - [x] Field level and page level validation errors (front-end and server-side) integrated and functioning.
 - [x] Verify that language for docket record for internal users and external users is identical.
 - [x] New screens have been added to pa11y scripts.
 - [x] All new functionality verified to work with keyboard and macOS voiceover https://www.apple.com/voiceover/info/guide/_1124.html.
 - [x] READMEs, other appropriate docs, and swagger/APIs fully updated.
 - [x] UI should be touch optimized and responsive for external only (functions on supported mobile devices and optimized for screen sizes as required).
 - [x] Interactors should validate entities before calling persistence methods.
 - [x] Code refactored for clarity and to remove any known technical debt.
 - [x] If new docket entries have been added as seed data to `efcms-local.json`, 3 local s3 files corresponding to that docketEntryId have been added to `web-api/storage/fixtures/s3/noop-documents-local-us-east-1`
 - [x] Acceptance criteria for the story has been met.
 - [x] If there are special instructions in order to deploy into the next environment, add them as a comment in the story.
 - [x] If the work completed for the story requires a reindex without a migration, or any other special deploy steps, apply these changes to the following flexion branches:
   - [ ] experimental1
   - [ ] experimental2
   - [ ] experimental3
   - [ ] experimental4
   - [ ] experimental5
   - [ ] experimental6
   - [ ] develop
 - [ ] Reviewed by UX on a deployed environment.
 - [ ] Reviewed by PO on a deployed environment. Can be deployed to the Court's `test` environment if prod-like data is required. Otherwise deployed to any `experimental` environment.
 - [ ] Deployed to the Court's `staging` environment.

**Comment by katiecissell** [comment]
UX Notes:
[Figma File](https://www.figma.com/file/Ua7LO6wxZ7oFmuwOT7T0Xa/USTC-%7C-Pending-Motions-added-to-Dashboard-%26-Activity-Report-Updates?type=design&node-id=73%3A20895&mode=design&t=qi1cV78ytZooMCdM-1)

Add three tabs below the trial session cards.
- tabs should be Recent Messages, Submitted/CAV (#entries), and Pending Motions (#entries).
- Recent messages should have a 'view all' link to the right above the table. (does not need a table counter since its limited to 5)
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBM2l3QVE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--3feb854b825d708763e1b25d18d4f3df22d8a3a3/image.png)
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBM3V3QVE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--007b780ef41eae2c2e685d51ba4fd769ea088c50/image.png)

 - Pending Motions should have text above the table: "Showing motions pending for more than 180 days. To view all, run the Pending Report." Pending Report text should be a link that links to the pending report page
- Table columns should include Docket # (with suffix and appropriate consolidated case icons), No. of Cases, Petitioners, "generic" title of motion type (as opposed to customized description seen on the docket record), # of days pending, Primary Issue, Final Brief Due Date, and Status of Matter
- Just like for CAV/Submitted users should be able to add/edit primary issue, Final Brief Due Date, and status of matter. (Should use modal identical to CAV/Submitted)
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBLysyQVE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--1fb239323dd423e112780dd40c0b14230c409175/image.png)


If there are no pending motions, the empty state should say "There are no motions pending for more than 180 days."
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBL2kyQVE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--15679c29bc87e2f690e44d4334f4a7185b90c2f7/image.png)

**Comment by ttlenard** [comment]
## Test Cases

#### 1) Judge user accesses their Dashboard; There are now 3 new tabs below the Trial Sessions boxes.

- Log in as a Judge User
- Review your Dashboard

Expected Results:
- [x] **NEW FUNCTIONALITY** - There are now 3 tabs below the Trial Session Boxes
- [x] **NEW FUNCTIONALITY** - Tabs are 
	- [x]  **NEW FUNCTIONALITY** - Recent Messages, this is the default tab  that is displayed
	- [x]  **NEW FUNCTIONALITY** - Submitted/CAV (# entries) is the second tab
	- [x]  **NEW FUNCTIONALITY** - Pending Motions (# entries) is the third tab

*Repeat this test with a Chambers user.


#### 2) Judge Use can view and respond to messages as per normal on the Recent Messages tab.

- This is a regression test
- Ensure that the functionality on this tab functions as it did prior to it being in a tab.

Expected Results:
- [x] Link to view all messages navigates user to the Messages page, where they can use the inbox, sent, completed tabs
- [x] User can click to view a message and respond appropriately
- [x] Normal display rules still apply (unread is bold, has an envelope, after being read it is no longer bolded, etc.)



#### 3) Judge user clicks on the Submitted/CAV tab; Functionality on this tab is the same as when this data was displayed in the Submitted/CAV Cases Section prior to this story.

- This is a regression test
- Ensure that the functionality in this new tab functions as it did prior to it being in a tab.

Expected Results:
- [x] All regression tests pass on this page
- [x] All regression tests pass to ensure that the data on this tab is also displayed correctly on the Activity report

*Repeat this test with a Chambers user



#### 4) Judge user clicks on the Pending Motions tab; Tab displays data on the page and in the table, similar to the mockup.

- Continuing from the previous test
- Click on the Pending Motions tab
- Review the page:

Expected Results:
- [x] **NEW FUNCTIONALITY** - There is text above the table that displays, "Showing motions pending for ~~more than~~ 180 days or more. To view all, run the Pending Report." 
- [x] **NEW FUNCTIONALITY** - Clicking on the pending report link navigates user to the Pending report
- [x] **NEW FUNCTIONALITY** - The total count at the top of the table matches the total count in the tab header
- [x] **NEW FUNCTIONALITY** - The total count matches the total entries in the table
- [ ] **NEW FUNCTIONALITY** - Column headers include:
	- [x] Docket No. 
	- [x] No. of Cases (if the case is part of a consolidated group, the total number of cases in the group will be displayed)
	- [x] Petitioner(s)
	- [x] Motion
	- [x] ~~No. of~~ Days Pending
	- [x] Final Brief Due Date
	- [x] Status of Matter
	- [x] Edit link is available for each row.
	- [x] Below the row with the Motion info is a section for Primary Issue (these will be blank initially)
- [x] **NEW FUNCTIONALITY** - If a case is part of a consolidated group, the consolidated icon will display next to the docket number

*Repeat this test with a Chambers user



#### 5) Judge user that has motions that have been pending for ~~more than~~ 180 days or more clicks on the Pending Motions tab; Clicks the edit link to input information; Data saves appropriately.

- Be sure that you log in as a Judge user that has motions that have been pending for ~~more than~~ 180 days or more (can check the Pending report to find a Judge).
- Click on the Pending Motions tab
- Click on the Edit link for one of the pending motions

Expected Results:
- [x] **NEW FUNCTIONALITY** - The Edit Details pop-up displays
- [x] **NEW FUNCTIONALITY** - Displays the Docket number and Petitioner Name
- [x] **NEW FUNCTIONALITY** - Final Brief Due Date field has a calendar icon, has usual date validation 
- [x] **NEW FUNCTIONALITY** - Status of Matter Dropdown with the following dropdown options
	- [x] Awaiting Consideration, Awaiting Briefs, Drafting, Reviewing draft, Submitted to the Chief Judge, Revising Draft, Submitted to Reporter, Stayed
- [x] **NEW FUNCTIONALITY** - Text area for Primary Issue
- [x] **NEW FUNCTIONALITY** - User can save without inputting any data, or save with a combination of fields (none are required for saving)
- [x] **NEW FUNCTIONALITY** - Data that is saved displays on the appropriate row in the Pending Motions tab.
- [x] **NEW FUNCTIONALITY** - Data that is edited and saved displays approprately on the pending motions tab.

*Repeat this test with a Chambers user



#### 6) Compare the data on the pending report with various judges

- Run the pending report for various judges and take note of a judge user that has motions pending for ~~more than~~ 180 days or more.
- Take a screen capture of all of the cases/motions that are in this state.
- Be sure to also note if there are pending items that are not motions, but have been pending for ~~more than~~ 180 days or more
- Log in as that Judge User
- Click on the Pending Motions tab
- Review the data

Expected Results:
- [x] **NEW FUNCTIONALITY** - The total number of Pending motions matches between the pending report and the Pending Motions tab on the dashboard
- [x] **NEW FUNCTIONALITY** - The motion title is correctly displayed for each motion
- [x] **NEW FUNCTIONALITY** - Pending items that are ~~over~~ 180 days or more that are not motions do not display on the pending motions tab on the Judges Dashboard.

*Repeat this test with multiple judges
*Repeat this test while logged in as various Chambers Admins



#### 7) Motion that is displayed on the Pending Motions tab get's removed from the tracked items tab on the case; Motion no longer displays on the Judges Dashboard view.

- Log in as a Judge that has motions pending for ~~more than~~ 180 days or more
- Take note of one of the cases and the specific motion name that has some data in the primary issue, status of matter, and final brief due date fields (can use the same motion from test case 5)
- Next, log in as a docket clerk
- Navigate to the case
- Click on the Tracked items tab and be sure you are on the Pending tab
- Remove the pending item
- Next, log in to DAWSON as the Judge user you started this test with
- Click on the Pending motions tab on your dashboard
- Review the results in the Pending motions table

Expected Results:
- [x] **NEW FUNCTIONALITY** - The pending motion that was previously displayed on the Pending motions tab is no longer present.

*Repeat this test with a Chambers user



#### 8) Motion that was previously displayed on the pending motions tab get's marked to add to the pending report again; Motion is available on the pending report and on the Pending Motions tab.

- Continuing from the previous test
- Log in as a docket clerk
- Navigate to the case where you removed the pending item in the previous test
- Click on the edit link on the docket record for the pending motion
- Check the box to "Add to Pending Report"
- Click Save 
- The Pending Report tab under tracked items should now display the pending item again
- Next, Log in as the Judge user you started this test with
- Click on the Pending motions tab on your dashboard
- Review the results in the Pending motions table

Expected Results:
- [x] **NEW FUNCTIONALITY** - The pending motion is displayed in the table again with the days in pending status based off of the filed date.
- [x] **NEW FUNCTIONALITY** - The data that was previously in the Primary Issue, Status of Matter, and Final Brief Due date fields is not retained, and the fields are blank again.

*Repeat this test with a Chambers user



#### 9) Docket clerk multidockets a paper filed motion across a consolidated group of cases assigned to a Judge;  Date filed is ~~more than~~180 days or more; Multi-docketed motion displays singly on the Judges Dashboard; multiple rows on pending report.

- Log in as a docket clerk
- Navigate to a lead consolidated case where the group is assigned to a judge
- Click on the Create button
- Select Paper filing
- Upload a document
- For filed date, select a date that is ~~more than~~ 180 days or more from today.
- For document type, select any motion
- Fill out the rest of the required fields and click Save and Serve
- Indicate that you would like to file and serve in all cases in the group
- Click done on the print for paper service
- Pending motion should be on the tracked items tab as well as the pending report
- Log in as the Judge user that is assigned to the case
- Review the Pending motions tab on your dashboard

Expected Results:
- [x] **NEW FUNCTIONALITY** - The Pending motion for the lead case is displayed on the Pending motions table
- [x] **NEW FUNCTIONALITY** - The Lead Consolidated Case icon is displayed to the left of the docket number, none of the other motions filed on the member cases are displayed in the table.
- [x] **NEW FUNCTIONALITY** - The No. of days Pending column displays the correct number of days based off of the filing date.
- [x] **NEW FUNCTIONALITY** - The Primary issue, Status of Matter, and Final brief due date fields are blank, but can be edited.
- [x] **NEW FUNCTIONALITY** - Data added to the Primary issue, Status of Matter, and Final Brief Due date fields can be saved appropriately.

*Repeat this test with a chambers user



#### 10) Docket Clerk files a paper motion singly in a member case in a consolidated group assigned to a Judge; Date filed is more than 180 days; Single motion displays on the Judges Dashboard; single row on pending report.

- Log in as a docket clerk
- Navigate to a member consolidated case where the group is assigned to a judge
- Click on the Create button
- Select Paper filing
- Upload a document
- For filed date, select a date that is ~~more than~~ 180 days or more from today.
- For document type, select any motion
- Fill out the rest of the required fields and click Save and Serve
- Click done on the print for paper service
- Pending motion should be on the tracked items tab as well as the pending report
- Log in as the Judge user that is assigned to the case
- Review the Pending motions tab on your dashboard

Expected Results:
- [x] **NEW FUNCTIONALITY** - The Pending motion for the member case is displayed on the Pending motions table
- [x] **NEW FUNCTIONALITY** - The member Consolidated Case icon is displayed to the left of the docket number.
- [x] **NEW FUNCTIONALITY** - The No. of days Pending column displays the correct number of days based off of the filing date.
- [x] **NEW FUNCTIONALITY** - The Primary issue, Status of Matter, and Final brief due date fields are blank, but can be edited.
- [x] **NEW FUNCTIONALITY** - Data added to the Primary issue, Status of Matter, and Final Brief Due date fields can be saved appropriately.

*Repeat this test with a chambers user



#### 11) Docket Clerk files a paper motion singly in a lead case in a consolidated group assigned to a Judge; Date filed is ~~more than~~ 180 days or more; Single motion displays on the Judges Dashboard; single row on pending report.

- Log in as a docket clerk
- Navigate to a lead consolidated case where the group is assigned to a judge
- Click on the Create button
- Select Paper filing
- Upload a document
- For filed date, select a date that is ~~more than~~ 180 days or more from today.
- For document type, select any motion
- Fill out the rest of the required fields and click Save and Serve
- Indicate that you would like to file and serve in only the Lead case in the group
- Click done on the print for paper service
- Pending motion should be on the tracked items tab as well as the pending report
- Log in as the Judge user that is assigned to the case
- Review the Pending motions tab on your dashboard

Expected Results:
- [x] **NEW FUNCTIONALITY** - The Pending motion for the lead case is displayed on the Pending motions table
- [x] **NEW FUNCTIONALITY** - The Lead Consolidated Case icon is displayed to the left of the docket number.
- [x] **NEW FUNCTIONALITY** - The No. of days Pending column displays the correct number of days based off of the filing date.
- [x] **NEW FUNCTIONALITY** - The Primary issue, Status of Matter, and Final brief due date fields are blank, but can be edited.
- [x] **NEW FUNCTIONALITY** - Data added to the Primary issue, Status of Matter, and Final Brief Due date fields can be saved appropriately.

*Repeat this test with a chambers user



#### 12) Docket Clerk files a paper motion in a case assigned to a Judge; Filed date is ~~180~~ 179 days from today; Pending motion does not display on the Judges Dashboard, Motion does display on the pending report.

- Log in as a docket clerk
- Navigate to a case that is assigned to a judge
- Click on the Create button
- Select Paper filing
- Upload a document
- For filed date, select a date that is exactly ~~180~~ 179 days from today.
- For document type, select any motion
- Fill out the rest of the required fields and click Save and Serve
- Click done on the print for paper service
- Pending motion should be on the tracked items tab as well as the pending report
- Log in as the Judge user that is assigned to the case
- Review the Pending motions tab on your dashboard

Expected Results:
- [x] **NEW FUNCTIONALITY** - Pending motion that is exactly ~~180~~ 179 days does NOT display on the Pending motions tab

*Repeat this test with a chambers user



#### 13) Docket Clerk files a paper motion in a case assigned to a Judge; Filed date is ~~181~~ 180 days from today; Pending motion displays on the Judges Dashboard, Motion is also displayed on the pending report.

- Log in as a docket clerk
- Navigate to a case that is assigned to a judge
- Click on the Create button
- Select Paper filing
- Upload a document
- For filed date, select a date that is exactly ~~181~~ 180 days from today.
- For document type, select any motion
- Fill out the rest of the required fields and click Save and Serve
- Click done on the print for paper service
- Pending motion should be on the tracked items tab as well as the pending report
- Log in as the Judge user that is assigned to the case
- Review the Pending motions tab on your dashboard

Expected Results:
- [x] **NEW FUNCTIONALITY** - Pending motion that is ~~181~~ 180 days displays on the Pending motions tab
- [x] **NEW FUNCTIONALITY** - The No. of days Pending column displays the correct number of days based off of the filing date.
- [x] **NEW FUNCTIONALITY** - The Primary issue, Status of Matter, and Final brief due date fields are blank, but can be edited.
- [x] **NEW FUNCTIONALITY** - Data added to the Primary issue, Status of Matter, and Final Brief Due date fields can be saved appropriately.

*Repeat this test with a chambers user




#### 13) Judge user that does not have any pending motions ~~over~~ 180 days or more views the Pending motions tab on their dashboard; No pending motions text displays appropriately.

- Log in as a Judge user that does not have any pending motions ~~over~~ 180 days or more
- Click on the Pending motions tab on your dashboard
- Review the page

Expected Results:
- [x] **NEW FUNCTIONALITY** - The text displays "There are no motions pending for ~~more than~~ 180 days or more."
- [x] **NEW FUNCTIONALITY** - Column data still display, as does the count (0), and the reference to the pending report.

**Comment by ttlenard** [comment]
Some testing feedback.  @nechama-krigsman @cruzjone-flexion @rachelschneiderman 

We're seeing some slow behavior when logged in as a judge/chambers user.  It's taking around 20 seconds to load some judge's dashboards, and when logging in as Judge Carluzzo, I get a 504 error (See screen grab).  
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBL0VuQWc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--b64a96ea3ceb743260496958d288e70f9fde653d/image.png)

**Comment by ttlenard** [comment]
@cruzjone-flexion @nechama-krigsman @rachelschneiderman 

Some testing feedback: (Note that 1, 3, and 4 are optional updates)

1. Test Case 2 - The column header for No. Days Pending is different than the mockup. After discussing in the slack UX channel, Chris is ok with how it is now, but would actually prefer "Days Pending" over the original mock and deployed column header.  Up to you all - if you can update this, "Days Pending" would be preferred. **FIXED**
2. Test Case 6 - The A/C states that the title of the motion will be the generic motion title, not the long title that is on the docket record.  I am seeing the entire motion title on the Pending motions tab, not the generic title. please fix.  **FIXED**
3. Not in the A/C or a specific test case, but it would be nice if when you click on the motion title, it opens in a new tab.  I could see this being a big convenience to the judge users. **FIXED**
4. Test Case 8 - Looks like the data that was added to the final brief due date, status of matter, and primary issue are being retained when the pending motion is added back to the pending report.  The notes in the story indicate that this data doesn't need to be retained. Dev Discretion on if you would like to wipe this data or not.  We are good either way. **FIXED**
5. Test Case 12/Test Case 3 - I updated the test cases to say 179 days, not 180 days. The A/C indicates that we display pending motions that are 180 days or more.  What we need to fix is the description at the top of the page so that it is accurate. Suggestion: "Showing motions pending for 180 days or more. To view all, run the Pending Report."**FIXED**
6. Test Case 13 - When there isn't any Pending motions for a judge, we need to update the text to state, "There are no motions pending for 180 days or more." so that we are consistent. **FIXED**

**Comment by ttlenard** [comment]
@cruzjone-flexion @nechama-krigsman @cruzjone-flexion 
More testing feedback:

I was testing something else this morning and I received some more 502 errors while logging in as the following Judges. Not sure at this point if other Judges are also getting the same error?

judge.copeland@example.com
judge.lauber@example.com
judge.holmes@example.com

This is the error I am getting for all of these users:

<img width="1510" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBOGM2QWc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--357805de8dd144967baab998fec7066f048d59f8/image.png" alt="image.png" />

**Comment by cruzjone-flexion** [comment]
Error 502 caused by payloads exceeding max size. Fixed and deployed

**Comment by ttlenard** [comment]
Comment ustaxcourt/ef-cms#6494 above isn't quite right.  

- Keep the days at 180. We only needed to change the informational text above the table.

Also, I spoke with Jonathon and he is going to do some additional testing by adding in 200 pending motions to a judge to ensure that it doesn't 502.

**Comment by cruzjone-flexion** [comment]
Updated days and label above table.
Also deployed to an experimental env with a multiplier of x100 on the endpoint and it returned 400 pending motions with no issues.

<img width="1277" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBMkk4QWc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--a5fd4b68dde3accea48430651867f258e99a80d8/image.png" alt="image.png" />

**Comment by ttlenard** [comment]
A bit more feedback.

I actually messed up my testing data earlier today, and accidentally put in the wrong filed date. As a docket clerk, I manually edited the document on the docket record and changed the filing date to a different date on a couple of entries. I thought it might just take ES to catch up with dynamo, but it's been more than a few hours and I am still seeing incorrect data on the pending motions tab.  See notes in the screen grab:
<img width="1455" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBK2RCQWc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--b4e310f57bc05eff1db4a29b17e4b07de8b7f0c0/image.png" alt="image.png" />
